main.js

```diff
@@ -59,7 +76,57 @@ function getCurrentGutter() {
 	return getGridProperties().gutter;
 }
 
+/**
+ * This sets MediaQueryListeners on all the o-grid breakpoints
+ * and fires a `o-grid.layoutChange` event on layout change.
+ */
+function addBreakpointListeners() {
+	try {
+		// Create a map containing all breakpoints exposed via html:before
+		const breakpoints = new Map(Object.entries(getGridBreakpoints().layouts));
+		const decr1 = val => `${Number(val.replace('px', '') - 1)}px`;
+
+		// Generate media queries for each
+		breakpoints.forEach((width, size) => {
+			const queries = Array();
```

Comment:
Could we change this from using the `Array` constructor to an array literal instead please. The reason for this would be consistency in our code across components.